pytket/extensions/quantinuum/backends/quantinuum.py

```diff
@@ -655,14 +657,16 @@ def rebase_pass(self) -> BasePass:
             allow_swaps=self.compilation_config.allow_implicit_swaps,
         )
 
-    def default_compilation_pass(self, optimisation_level: int = 2) -> BasePass:
+    def default_compilation_pass(
```

Comment:
should also document the `timeout` parameter

Reply:
done